plugins/woocommerce-admin/package.json

```diff
@@ -33,21 +33,14 @@
 		"prepack": "pnpm install && pnpm run lint && pnpm run test && cross-env WC_ADMIN_PHASE=core pnpm run build",
 		"packages:fix:textdomain": "node ./bin/package-update-textdomain.js",
 		"packages:watch": "cross-env WC_ADMIN_PHASE=development pnpm run:packages -- start --parallel",
-		"pre-release": "./bin/pre-release.sh",
-		"publish-packages:check": "pnpm run build:packages && pnpm publish --dry-run --filter ../../packages/js/ --publish-branch main --report-summary && cat ../../pnpm-publish-summary.json && rimraf ../../pnpm-publish-summary.json",
-		"publish-packages:dev": "pnpm run build:packages && pnpm publish --filter ../../packages/js/ --publish-branch main --tag next",
-		"publish-packages:prod": "pnpm run build:packages && pnpm publish --filter ../../packages/js/ --publish-branch main",
-		"reformat-files": "wp-scripts format-js -- --ignore-path .eslintignore",
```

Comment:
@psealock I removed the `publish-packages` scripts, do you use these still? (I know you released them last). I kinda figured that was not the case, but want to double check.

Reply:
Ah yes, thanks @louwie17. Thats right, we are now using [automation](https://github.com/woocommerce/woocommerce/pull/33922) from the new Package Release tool.